pac4j-core/src/main/java/org/pac4j/core/client/IndirectClient.java

```diff
@@ -88,6 +127,16 @@ public final RedirectAction getRedirectAction(final WebContext context) throws H
     protected abstract RedirectAction retrieveRedirectAction(final WebContext context) throws HttpAction;
 
     /**
+     * Retrieve the redirect action for the logout.
+     * 
+     * @param context the web context
+     * @return the redirection action
+     */
+    protected RedirectAction retrieveLogoutRedirectAction(final WebContext context) {
```

Comment:
Well that should be refactored then. But not all indirect clients should perform a logout redirect. How would you do this ?